### DISCUSSION TITLE
createMany is not existent for my model definition

### DISCUSSION THREAD

**Discussion by oof2win2** [prisma/prisma#10705] [Q&A] [answered]
Hey there. I am trying to get `createMany` working on my model `Actions`. Below is my database definition. As [the video in the docs says](https://www.prisma.io/docs/concepts/components/prisma-client/crud#create-multiple-records), i need to add `createMany` to the `previewFeatures`, but it instead tells me that it is already included. I can't however seem to use it, and in `node_modules/.prisma/client/index.d.ts`, there are two references to `createMany`: the type `Prisma.PrismaAction` and a comment. Am I doing something wrong? When I switch the DB provider to `mysql`, the `createMany` methods get generated. Is this another exception to SQLite?
```
generator client {
  provider = "prisma-client-js"
  previewFeatures = ["createMany"]
}

datasource db {
  provider = "sqlite"
  url      = env("DATABASE_URL")
}

model BotConfig {
  guildID                   String    @id
  apikey                    String?
  owner                     String
  lastNotificationProcessed DateTime  @default(now()) @updatedAt
  actions                   Actions[]
}

model InfoChannel {
  id        Int    @id @default(autoincrement())
  channelID String @unique
  guildID   String
}

model Command {
  id      String @id
  guildID String
  name    String
}

model Actions {
  id          Int       @id @default(autoincrement())
  guildID     String    @unique
  report      String    @default("none")
  revocation  String    @default("none")
  botConfigID String
  config      BotConfig @relation(fields: [botConfigID], references: [guildID])
}
```

**Comment by oof2win2** [comment, marked as answer]
Nevermind, it isn't supported [according to the docs](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#remarks-9)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
